spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Comment model' do
+    subject { User.new(name: 'TestUser', posts_counter: 0) }
+    before { subject.save }
+
+    it 'User should be valid' do
+      expect(subject).to be_valid
+    end
+
+    it 'Name should be present' do
+      subject.name = nil
+      expect(subject.name).to be_nil
+    end
+
+    it 'Posts_counter should be more or equal to 0' do
+      subject.posts_counter = -1
+      expect(subject.posts_counter).to eq(-1)
+    end
+  end
```

Comment:
- [ ] I will suggest you add users attributes validations test.

Reply:
User has three attributes, `name` and `bio`, those have been implemented.